portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,13 +19,24 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Samuel!</h1>");
+    response.setContentType("text/json;");
+   // response.getWriter().println("Hello Samuel!");
```

Comment:
Please delete the the commented line.  You might need to comment some code when debugging, but commented code can't be submitted or merged into master branch/production code base.

Reply:
Done. Thanks